package.json

```diff
@@ -78,6 +78,7 @@
     "reflect-metadata": "^0.1.13",
     "replace-in-file": "^4.1.1",
     "semver": "^5.6.0",
+    "snarkdown": "^1.2.2",
```

Comment:
didn't we already have a lib to render md in the admin  ? if no, snarkdown is a great pick! its soo lean !